lib/Models/user.dart

```diff
@@ -1,6 +1,20 @@
 import 'package:cloud_firestore/cloud_firestore.dart';
 
 class UserClass {
+
+  UserClass(this.username, this.height, this.weight, this.age, this.email,
+      this.firstname, this.lastname, this.system, this.userID);
+
+  UserClass.fromJson(Map<String, Object?> user)
+      : this.username = user['username']! as String,
+        this.height = user['height']! as double,
+        this.weight = user['weight']! as double,
+        this.age = user['age']! as int,
+        this.email = user['email']! as String,
+        this.firstname = user['firstname']! as String,
+        this.lastname = user['lastname'] as String,
+        this.system = user['system']! as String,
+        this.userID = user['userID']! as String;
```

Comment:
add comma's to the constructors and spacing between the constructor and the params for the rest of the models. else all the comments will be about this haha